README.md

```diff
@@ -1,3 +1,71 @@
-﻿# KMD Logic Email Api Client
+﻿# KMD Logic Email Api Client ![Email icon](Email.png)
 
 A dotnet client library for using Email module in KMD Logic platform through API.
+## The purpose of the Email API
+
+To allow products using the Logic platform to **send email using custom email domain address**  
+
+## EmailClient
+
+The Logic EmailClient provides options to:
+* Upload Attachment
```

Comment:
Attachment, template, etc should be in plural I guess. Like `Attachments`, `Templates`, etc.

Reply:
Resolved